### DISCUSSION TITLE
Assistance Needed for Hosting Gemini-Powered Chatbot on Vercel

### DISCUSSION THREAD

**Discussion by SaketGopaluni** [vercel/community#7306] [Feature Question]
### Question

Hi,

I have been working on Vercel to host a Gemini-powered chatbot using one of your templates. However, the provided use case for booking flights does not align with my requirements, so I attempted to modify the SDK accordingly. Unfortunately, I have encountered errors while running the code on my Windows laptop, preventing me from proceeding with the hosting.

One specific error that I encountered is as follows:

✓ Ready in 2.1s
 ○ Compiling /middleware ...
 ✓ Compiled /middleware in 674ms (79 modules)
 ○ Compiling / ...
 ⨯ ./lib/chat/actions.tsx:30:0
Module not found: Package path ./google is not exported from package C:\Users\saket\gemini-chatbot\node_modules\ai (see exports field in C:\Users\saket

Additionally, I have purchased a domain and am eager to host the chatbot, but these issues are hindering my progress. Could you please assist me in resolving these errors and guide me on how to edit the code for my specific use case? Access to the necessary instructions or assistance from the GitHub page would be greatly appreciated.

**Comment by pawlean** [comment]
Hi @SaketGopaluni!

> Module not found: Package path ./google is not exported from package C:\Users\saket\gemini-chatbot\node_modules\ai (see exports field in C:\Users\saket

The error message indicates that there's an issue with importing a module from the ai package. Looks like the google package is not exported. Can you check that the packages are installed correctly?

[a comment by an automated account is not shown]
